layers/meta-tegrademo/recipes-bsp/tegra-fitimage/tegra-fitimage_1.0.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Install tegra fitImage in the location expected by u-boot"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+COMPATIBLE_MACHINE = "(cot)"
+
+do_install() {
+    fitImgPath=$(readlink -f ${DEPLOY_DIR_IMAGE}/fitImage)
+    fitImgName=$(basename "$fitImgPath")
+    install -d ${D}/boot
+    install -m 0644 $fitImgPath ${D}/boot/
+    ln -sf $fitImgName ${D}/boot/fitImage
+}
+
+ALLOW_EMPTY_${PN} = "1"
+
+do_install[depends] += "linux-tegra:do_deploy"
+FILES_${PN} = "/boot/fitImage*"
+PROVIDES = "${PN}"
```

Comment:
You shouldn't have to set PROVIDES here - '${PN}' is the default for every recipe.